app/_how-tos/run-docker-images.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Run Docker images
+description: "Run docker images"
+
+products:
+    - gateway
+
+content_type: how_to
+
+tldr: 
+  q: How do I build a custom {{site.base_gateway}} Docker image?
+  a: Create a Docker file
+
+works_on:
+    - on-prem
+
+breadcrumbs:
+    - /gateway/
+min_version:
+  gateway: '3.4'
+prereqs:
+  skip_product: true 
+  inline:
+    - title: Download the {{site.base_gateway}} entry-point script.
+      content: |
+        1. Download the {{site.base_gateway}} [entry-point script](https://raw.githubusercontent.com/Kong/docker-kong/master/docker-entrypoint.sh):
+            ```sh
+            curl -O https://raw.githubusercontent.com/Kong/docker-kong/master/docker-entrypoint.sh
+            ```
+        2. Make the script executable
+            ```sh
+            chmod +x docker-entrypoint.sh
+            ```
+    - title: Download the {{site.base_gateway}} base image
+      content: |
+        1. [Download](/gateway/install/#linux) the image for your desired operating system.
+        2. Rename the file to either `kong.deb` or `kong.rpm` depending on the package.
+---
+
+@TODO https://docs.konghq.com/gateway/latest/install/docker/
```

Comment:
To answer your question: "Let me know if you feel like the doc requires that second part I showed you and I’ll write the other doc"

I think having a clear link to this doc is fine since like you said, people may want the custom image but not the Gateway/database container. I think I'd like to see this how to cover the database (marked optional) and then running the KG container like you demoed.
```suggestion
@TODO https://docs.konghq.com/gateway/latest/install/docker/

Include:
- how to optionally start the database
- how to install Kong from the image
```